helm/robusta/kubewatch.yaml

```diff
@@ -1,4 +1,3 @@
-namespace: ""
```

Comment:
Why did you remove it?
What was the purpose of that?

Reply:
i moved it to the kubewatch-configmap.yaml file so i can use the release value, the content of the file was used for the config map.